test/helpers/kubectl.go

```diff
@@ -2413,7 +2413,9 @@ func (kub *Kubectl) overwriteHelmOptions(options map[string]string) error {
 		options["imagePullSecrets[0].name"] = config.RegistrySecretName
 	}
 
-	if CiliumEndpointSliceFeatureEnabled() {
+	if _, found := options["enableCiliumEndpointSlice"]; !found &&
+		CiliumEndpointSliceFeatureEnabled() {
```

Comment:
IIRC, the K8s>=1.21 was set arbitrarily, to not run everywhere in CI. So this helper can now be removed since it's disabled by default anyway. Do we even have any tests running with `enableCiliumEndpointSlice`?

Reply:
Previous discussion was at https://github.com/cilium/cilium/pull/17698#discussion_r736766323.

Reply:
- The 4.9 job runs K8s 1.16 => CES disabled.
- The 4.19 runs K8s 1.22 => CES enabled for all.
- The net-next runs K8s 1.23 => CES enabled for all except for the Egress GW suite.

Do I miss something?

Reply:
Oh, no. I'm the one who missed something :facepalm: 
Thanks!